pom.xml

```diff
@@ -89,7 +89,7 @@
         <!-- align with org.apache.hbase:hbase -->
         <jackson1.version>1.9.13</jackson1.version>
         <!-- align with org.apache.spark:spark-core_2.12 -->
-        <jackson2.version>2.12.5</jackson2.version>
+        <jackson2.version>2.13.0</jackson2.version>
```

Comment:
This might cause a problem for Spark traversal since it is not aligned with spark-core_2.12 now.